src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,70 @@
-export const Tabs = () => {};
+import React, { useState } from 'react';
+
+export interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const [toggleState, setToggle] = useState(tabs[0]);
```

Comment:
you store `selectedTab` here, so rename the variable accordingly

Reply:
And you should save `id` in the `App` component:
![image](https://github.com/mate-academy/react_tabs/assets/86923089/5ddd4dfe-6008-4dad-9ddb-e6d9f6bd9ff1)